README.md

```diff
@@ -1,45 +1,53 @@
 # Gravity Web UI packages monorepo
+
 !["Gravity UI Web monorepo" banner image](./docs/gravity-ui-web-repo-img.png)
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/buildit/gravity-ui-web.svg)](https://greenkeeper.io/)
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 
-[Lerna](https://lernajs.io/) monorepo containing **Gravity design system** NPM packages relating to Web UIs:
-
-* [Gravity UI Web](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): The core SASS/CSS library used by all Gravity-powered web projects.
-* [Gravity UI Nunjucks](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): Nunjucks HTML templates for Gravity UI components. Also used to generate our [living pattern library](http://style.buildit.digital/).
-* Custom Semantic Release helpers:
-  * [Lerna Config](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/config)
-  * [Lerna Analyzer](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/lerna-analyzer)
-  * [Lerna Release Notes Generator](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/release-notes-generator)
+[Lerna](https://lerna.js.org/) monorepo containing **Gravity design system** NPM packages relating to Web UIs:
 
+- [Gravity UI Web](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): The core SASS/CSS library used by all Gravity-powered web projects.
+- [Gravity UI Nunjucks](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): Nunjucks HTML templates for Gravity UI components. Also used to generate our [living pattern library](http://style.buildit.digital/).
+- Custom Semantic Release helpers:
+  - [Lerna Config](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/config)
+  - [Lerna Analyzer](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/lerna-analyzer)
+  - [Lerna Release Notes Generator](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/release-notes-generator)
 
 ## Development
 
 ### Initial setup
+
 After an initial clone, or if any of the packages' dependencies have changed you should run the following **in the repo's root directory**:
 
 ```
 npm install
-npm lerna:bootstrap
+lerna bootstrap
```

Comment:
Actually, the instructions were correct as they were.

`lerna bootstrap` would only work if you happen to install Lerna globally, which I'd be reluctant to force people to do (I find global NPM things just clutter up your computer and sometimes lead to weird version conflicts when they differ to a locally installed copy in a project)